.github/workflows/publish-docker.yml

```diff
@@ -27,8 +29,12 @@ jobs:
         uses: docker/metadata-action@v3
         with:
           images: getmeili/meilisearch
+          # Output 'latest' tag only when 'release' happen, instead of when 'push tag' event happend (default behavior).
+          # See https://github.com/docker/metadata-action#latest-tag
           flavor: latest=false
-          tags: type=ref,event=tag
+          tags: |
+            type=ref,event=tag
+            type=raw,value=latest,enable=${{ github.event_name == 'release' && github.event.action == 'released' }}
```

Comment:
This should output a `latest` tag when `release` event happen, according to [Latest tag](https://github.com/docker/metadata-action#latest-tag):     

> _For conditionally tagging with latest for a specific branch name, e.g. if your default branch name is not master, use type=raw with a boolean expression:_
>
> ```
> tags: |
>   # set latest tag for master branch
>   type=raw,value=latest,enable=${{ github.ref == format('refs/heads/{0}', 'master') }}
> ```

Reply:
Not sure `github.event.action == 'released'` is useful, maybe I'm wrong?

Reply:
Yes you're right, I believe only `github.event_name === 'release'` is sufficient in this case. I included the `github.event.action === 'released'` only for explicity.